src/common/components/ui/WizardFooter.tsx

```diff
@@ -92,6 +101,22 @@ export const WizardFooter: React.FC<WizardFooterProps> = ({
               </Text>
             </ActionListItem>
           )}
+          <SplitItem isFilled />
+          {cluster && (
+            <ActionListItem>
+              <EventsModalButton
+                id="cluster-events-button"
```

Comment:
This class: src/ocm/components/clusterWizard/ClusterWizardStepHeader.tsxcould be deleted.

Reply:
In case of src/ocm/components/clusterDetail/ClusterDetail.tsx and src/ocm/components/clusterDetail/ClusterDetailsButtonGroup.tsx I can test it and the button is not duplicated.

Reply:
The only question is for src/ocm/components/AddHosts/AddHosts.tsx class. I don't know if of Day 2 part.

Reply:
Ok, I can test it and is working well. Now, I'll create a new component to mantain the same styles in all the pages.